README.md

```diff
@@ -98,4 +100,41 @@ We can extend the concept of the `as` syntax by generalizing the right hand side
 import value from "module" as {key1: "value1", key2: [1, 2, 3]};
 ```
 
-This would allow module attributes to scale up to multiple pieces of data, not privileging a single attribute.
+This would allow module attributes to scale to support a variety of metadata. It is currently unclear if the generalized constant form will be pursued in this proposal or in a follow up. In either case the addition of the generalized constant form is not hard to implement on top of the existing [spec text](https://tc39.github.io/proposal-module-attributes/). 
```

Comment:
s/implement/design, specify and implement/

Reply:
"existing spec text outline" (the spec text is not done)

In TC39 conventions, "implementation" happens in languages like C++; we don't tend to use that word for specification work

Reply:
fixed